LDMP/calculate_lc.py

```diff
@@ -89,7 +89,7 @@ def calculate_on_GEE(self):
             "crs": self.aoi.get_crs_dst_wkt(),
             "crosses_180th": crosses_180th,
             'legend_nesting': LCLegendNesting.Schema().dump(
-                self.lc_setup_widget.aggregation_dialog.nesting
+                self.lc_setup_widget.aggregation_dialog._nesting
```

Comment:
I think this should now be referencing `lc_setup.get_lc_nesting()`, no?

Reply:
I think it should revert to the previous one i.e. 
```
self.lc_setup_widget.aggregation_dialog.nesting
```

Unintended consequence of auto-refactoring as I did not modify this part.